### DISCUSSION TITLE
SDK .NET CORE

### DISCUSSION THREAD

**Discussion by biapar** [directus/directus#7804] [Archive]
Hi,

Is the an official Directus v9 SDK for .NET Core? I found this one https://github.com/kipergil/sdk-dotnet and only one fork.
How to generate a stub from Directus Rest APi?
Thanks.

**Comment by rijkvanzanten** [comment]
Hi there! There's no official .NET SDK at this point.

**Reply by biapar** [reply]
How to generate it for the v9 like that repo ?

**Reply by rijkvanzanten** [reply]
I don't know, we don't maintain that repo. From what I can tell, it's generated using Swagger (https://swagger.io/tools/swagger-codegen/) so you might be able to use [the OpenAPI spec endpoint](https://docs.directus.io/reference/api/system/server/#get-openapi-specification) to get the spec, and feed that to the swagger generator.
